systers_portal/blog/tests/test_forms.py

```diff
@@ -12,8 +13,11 @@ class AddNewsFormTestCase(TestCase):
     def setUp(self):
         self.user = User.objects.create_user(username='foo', password='foobar')
         self.systers_user = SystersUser.objects.get(user=self.user)
+        country = Country.objects.create(name='Bar', continent='AS')
```

Comment:
@SanketDG . Yes I am loading all the communities and then searching based on the the query string. You could check my latest PR. It would clear all doubts IG. :)